docs/about/features_index/fed_eval.rst

```diff
@@ -7,12 +7,12 @@ Federated Evaluation
+Model evaluation is an essential part of the machine learning development cycle. In a traditional centralized learning system, all evaluation data is collected on a localized server. Because of this, centralized evaluation of machine learning models is a fairly straightforward task. However, in a federated learning system, data is distributed across multiple decentralized devices or nodes. In an effort to preserve the security and privacy of the distributed data, it is infeasible to simply aggregate all the data into a centralized system. Federated evaluation offers a solution by assessing the model at the client side and aggregating the accuracy without ever having to share the data. This is crucial for ensuring the model's effectiveness and reliability in diverse and real-world environments while respecting privacy and data locality.
 
 OpenFL's Support for Federated Evaluation
 -------------------------------------------------
 
-OpenFL, a flexible framework for Federated Learning, has the capability to perform federated evaluation by modifying the federation plan. In this document, we will show how OpenFL can facilitate this process through its task runner API (aggregator-based workflow), where the model evaluation is distributed across various collaborators before being sent to the aggregator. For the task runner API, this involves minor modifications to the ``plan.yaml`` file, which defines the workflow and tasks for the federation. In particular, the federation plan should be defined to run for one forward pass and perform only aggregated model validation
+OpenFL, a flexible framework for Federated Learning, has the capability to perform federated evaluation by modifying the federation plan. In this document, we will show how OpenFL can facilitate this process through its task runner API (aggregator-based workflow), where the model evaluation is distributed across various collaborators before being sent to the aggregator. For the task runner API, this involves minor modifications to the ``plan.yaml`` file, which defines the workflow and tasks for the federation. In particular, the federation plan should be defined to run for one forward pass and perform only aggregated model validation.

```

Comment:
I would simply refer to Task Runner API in this paragraph. Aggregator-based workflow is an evolution of the Task Runner API, which is only at an experimental stage right now.

Reply:
The "aggregator-based workflow" nomenclature is something we want to get away from. This was the old name for the TaskRunner API ([then](https://github.com/securefederatedai/openfl/tree/v1.4?tab=readme-ov-file#getting-started) vs. [now](https://github.com/securefederatedai/openfl/tree/v1.7?tab=readme-ov-file#features)).